packages/react/src/components/SignOutButton.tsx

```diff
@@ -23,7 +22,7 @@ export const SignOutButton = withClerk(
     };
 
     const wrappedChildClickHandler: React.MouseEventHandler = async e => {
-      await safeExecute((child as any).props.onClick)(e);
+      await safeExecute(child.props.onClick)(e);
```

Comment:
There's a type error originating from this change 👀

Reply:
it's caused by the vscode `source.fixAll.eslint` setting on `codeActionsOnSave` :( . Nice catch!